pkg/middlewares/forwardedheaders/forwarded_header.go

```diff
@@ -42,10 +72,100 @@ func (x *XForwarded) isTrustedIP(ip string) bool {
 	return x.ipChecker.IsAuthorized(ip) == nil
 }
 
+// clean up IP in case it is ipv6 address and it has {zone} information in it,
+// like "[fe80::d806:a55d:eb1b:49cc%vEthernet (vmxnet3 Ethernet Adapter - Virtual Switch)]:64692"
+func ipv6fix(clientIP string) string {
```

Comment:
Can we rename this to `removeIpv6Zone`? It is more accurate.

Reply:
done